lib/dest/write-contents/index.js

```diff
@@ -38,26 +40,13 @@ function writeContents(opt) {
     // This is invoked by the various writeXxx modules when they've finished
     // writing the contents.
     function onWritten(writeErr) {
-      if (isErrorFatal(writeErr)) {
+      if (fo.isOverwriteError(writeErr, file.flag)) {
```

Comment:
Yeah, now that you mention it, I agree. I took the "direction" of the original `isFatalError` functions but with the added specificity of the reasons (overwrite, unlink) it is kind of awkward. How about `emitOverwriteError` and `emitUnlinkError` or something like that?